db/models.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Date,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+from db.database import Base
+
+
+class Book(Base):
+    __tablename__ = "books"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), unique=True, nullable=False)
+    summary = Column(String(1023))
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"), nullable=False)

```

Comment:
Ensure that the `author_id` foreign key constraint is correctly set up in the database to reference the `id` column of the `author` table. This is crucial for maintaining referential integrity.